R/truncation.R

```diff
@@ -0,0 +1,130 @@
+#' Adjust or subset a line list to account for right-truncation
+#'
+#' @description
+#' Adjust or subset the line list `<data.frame>` by either changing dates that
+#' are within the right-truncation window to `NA` or removing cases that have
+#' occurred within a delay period. This occurs to recent cases or outcomes
```

Comment:
reading this I'm not sure what it means - what is the right-truncation window? And the delay period? Perhaps say explicitly that a random delay is added to a chosen event and if this is after a cut-off date will be either removed or will have all their dates set to NA?